src/statsmodels.jl

```diff
@@ -0,0 +1,8 @@
+# Note both predict and RegressionModel are defined in StatsBase
+@recipe function f(mod::RegressionModel)
+    newx = [ones(200) range(extrema(mod.model.pp.X[:,2])..., length = 200)]
```

Comment:
So the interface being `name` can be a symbol, but also a Tuple or Vector of symbols, length, and then the plot being bivariate or 3d depending on that, and the regression keeping all the other input variables at their mean?

Reply:
I just meant that the second column isn't always going to be what we want to plot.

For example...
```julia
mod = lm(@formula(PetalLength ~ 0 + PetalWidth), iris)
```
would get rid of the intercept so the 2nd column wouldn't even exist. I was just trying to find something that would ensure the right coefficient is grabbed from the model matrix.

Reply:
Yes I know and I agree, I was just trying to see if we could make it even more general

Reply:
That would actually be really convenient to be able to add multiple coefficients and see how they are interacting visually! I think this would also make for a good long term user interface when the stats ecosystem will adopt a better way of extracting information from models using coefficient names.

Reply:
`mod.model.pp.X` is specific to GLM, you should rather use `modelmatrix(mod)` instead.